application/features/bootstrap/MfaContext.php

```diff
@@ -126,6 +155,50 @@ public function theUserDoesNotHaveAManagerEmailAddress()
         $this->theResponseStatusCodeShouldBe(200);
     }
 
+    /**
+     * @Given the user has requested a new webauthn MFA
+     */
+    public function theUserHasRequestedANewWebauthnMfa()
+    {
+        $user = User::findOne(['employee_id' => $this->tempEmployeeId]);
+        Assert::notEmpty($user, 'Unable to find that user.');
+        $this->setRequestBody('type', Mfa::TYPE_WEBAUTHN);
+        $this->iRequestTheResourceBe('/mfa', 'created');
+
+        $id = $this->getResponseProperty('id');
+        Assert::notEmpty($id, 'Unable to get id of new Webauthn MFA');
+        $mfa = Mfa::FindOne(['id'=>$id]);
+        Assert::notEmpty($mfa, 'Unable to find that MFA.');
+
+        // It is too complicated at this point to come up with completely correct values
+        // These should get as far as producing a 400 status code with
+        // "error":"unable to create credential: Error validating challenge"
+
+        // These values are from values produced in serverless-mfa-api-go/webauthn_test.go Test_FinishRegistration
+        $reqValue = [
```

Comment:
Since this will be refactored to be an obviously invalid set of values (at least partially), if there's a way to make it clear when reading the code that some piece of it is invalid, that could be helpful. I'm not sure of a good way to do that, though.